_includes/asg/px-asg-intro.md

```diff
@@ -0,0 +1,3 @@
+## Portworx in an Auto Scaling Group
```

Comment:
nit: you can remove the heading levels (`##`) from the common include files since the files which include these might be at different heading levels and let's have those files control at what heading level this needs to be displayed.

Reply:
done!